src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -59,10 +60,14 @@ class GroupDetails extends React.Component<Props, State> {
     this.fetchData();
   }
 
-  componentDidUpdate(_prevProps, prevState: State) {
+  componentDidUpdate(prevProps: Props, prevState: State) {
     if (prevState.loading && !this.state.loading) {
       callIfFunction(this.props.finishProfile);
     }
+
+    if (prevProps.isGlobalSelectionReady !== this.props.isGlobalSelectionReady) {
+      this.fetchData();
+    }
```

Comment:
This view will not make a request until store is ready, so we need to check for this update so that we can make the request.